isis/src/control/objs/BundleResults/BundleResults.cpp

```diff
@@ -1164,6 +1318,27 @@ namespace Isis {
    */
   int BundleResults::numberObservations() const {
     return m_numberObservations;
+    // return m_numberImageObservations + m_numberLidarImageObservations; // TODO uncomment after changing numberObservations
```

Comment:
Is this coming in an upcoming commit?

Reply:
Once we get the BundleAdjust code changed to set the image and lidar observations separate, this will change. For right now, this preserves current functionality.

So this will change in the BundleAdjust PR, i just want the code here so it's ready.